src/components/app/details/appDetails/IssuesCard.tsx

```diff
@@ -31,7 +31,7 @@ const IssuesCard = ({ appStreamData, cardLoading, setErrorsList, toggleIssuesMod
     const [forceDeleteDialogMessage, setForceDeleteDialogMessage] = useState('')
     const [isImagePullBackOff, setIsImagePullBackOff] = useState(false)
 
-    const conditions = useMemo(() => appStreamData?.result?.application?.status?.conditions || [], [appStreamData])
+    const conditions = useMemo(() => appConditions || [], [appConditions])
```

Comment:
We can use appConditions directly. Memoization is not required here.

Reply:
done